### ISSUE TITLE
Full odds shiny ico

### ISSUE THREAD

**Issue by Gardenette** [untamed-team/project-untamed#1382]
Create a shiny icon for full odds Shinies and create a plugin that detects whether there is any odds increases like shiny charm involved

**Comment by LowEffortCoding** [comment]
i dont think this is necessary, like at all, but you could use the super shiny flag for it

**Comment by LowEffortCoding** [comment]
https://github.com/untamed-team/project-untamed/commit/fb8cb9e78a780722ef215f8c90fddc5cb9e6f2dc
